### DISCUSSION TITLE
How to use python to calculate the zodiac boundaries based on IAU's definition?

### DISCUSSION THREAD

**Discussion by BahnAstro** [Stellarium/stellarium#3127] [Q&A]
Hi, I reviewed Stellarium that it had make reference to IAU for zodiac boundaries, can I know how can I utilize the data in Github to calculate the actual zodiac boundaries based on IAU's definition?

**Comment by gzotti** [comment]
I am not aware of distinct Zodiac boundaries. The 88 IAU constellations are available in two versions: border definition or identification of constellation for a point. Both have been taken from reference publications. You can study Stellarium's code or develop your own ideas, but no, we don't provide additional free tutorials.
